test/test_mysql.py

```diff
@@ -32,10 +34,10 @@ def setUp(self):
         self.db = apel.db.apeldb.ApelDb('mysql', 'localhost', 3306, 'root', '',
                                         'apel_unittest')
 
-    # This method seems to run really slowly on Travis CI
-    #def tearDown(self):
-    #    query = "DROP DATABASE apel_unittest;"
-    #    subprocess.call(['mysql', '-u', 'root', '-e', query])
+    def tearDown(self):
+        # Delete to avoid unexplained hangs in setUp when dropping database
+        #
```

Comment:
```suggestion
        # Avoid unexplained hangs in setUp when dropping database.
```

I won't pass judgement on whether this is a good thing to do or not.